src/Select.js

```diff
@@ -1325,7 +1325,7 @@ Select.defaultProps = {
 	openOnClick: true,
 	optionComponent: Option,
 	pageSize: 5,
-	placeholder: 'Select...',
+	placeholder: 'Select',
```

Comment:
Please change the placeholder only for you unique input, this affects the other elements as well

| Before | After |
| --- | --- |
| ![screen shot 2018-11-26 at 10 46 37 am](https://user-images.githubusercontent.com/1847316/49028756-c17eb700-f168-11e8-9417-b6e356b659a1.png) | ![screen shot 2018-11-26 at 10 46 42 am](https://user-images.githubusercontent.com/1847316/49028688-985e2680-f168-11e8-8fce-190d9401df4f.png) |

Reply:
Yeah, I talked to UX team about that and they are expecting to do that change on all of the selects